### PR TITLE
Bug/segun/user dashboard

### PR DESCRIPTION
**Pull request by Segun-Ade**
#### What does this PR achieve?
This PR implements some of the feedbacks made by the mentors on the user dashboard

### CODE REVIEW

File: src/components/userDashboard/components/MainLayout.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            defaultSelectedKeys={['1']}
+            mode="inline"
+            style={{ margin: '5rem 0' }}
+            theme="light"
+        >
+            {DASHBOARD_MENU_STRING.map(menu => {
+                const {
+                    key, iconType, text, link,
+                } = menu;
+                const linkage = `${path}/${link}`;
```
Comment (enyeInc) [inline src/components/userDashboard/components/MainLayout.jsx]: have this linkage be a part of the menu object

File: src/components/userDashboard/components/Tables.jsx [hunk 2]
```diff
@@ -253,207 +180,109 @@ const topVendors = [
...
-            </Card>
+        <div>
+            <Col xs={22} md={24}>
+                <Card className="dashboard-cta hero-box">{Cta(path)}</Card>
+            </Col>
+
             {serviceTable}
-            <Card className="-card-fluid">{savedAddress}</Card>
-        </Col>
+        </div>
```
Comment (enyeInc) [inline src/components/userDashboard/components/Tables.jsx]: use a fragment to encapsulate this piece of code

File: src/components/userDashboard/components/style.scss [hunk 3]
```diff
@@ -29,36 +29,48 @@
 .-light {
     font-weight: 300;
     display: inline-block;
-    font-size: 0.8rem;
+    // font-size: 0.8rem;
```
Comment (enyeInc) [inline src/components/userDashboard/components/style.scss]: remove

File: src/components/userDashboard/components/style.scss [hunk 4]
```diff
@@ -29,36 +29,48 @@
...
+    // @media only screen and (min-width: 75rem) {
+    //     width: calc(35rem + 500 * (100vw - 1280px)/1300);
+    // }
 }
 
 @media only screen and (min-width: 45rem) {
     .-light {
         margin-right: 10px;
-        padding: 10px;
+        // padding: 10px;
```
Comment (enyeInc) [inline src/components/userDashboard/components/style.scss]: remove all the unused code

File: src/components/userDashboard/components/Tables.jsx (hunk 2 above)
Comment (Segun-Ade) [reply src/components/userDashboard/components/Tables.jsx]: I did use a fragment initially, but it was resulting in the below linting errors:
`Fragments are only supported starting from React v16.2. Please disable the `react/jsx-fragments` rule in ESLint settings or upgrade your version of React.eslint(react/jsx-fragments)`

Comment (edbella) [reply src/components/userDashboard/components/Tables.jsx]: @Segun-Ade I think you can disable the eslint rule here.

Comment (Segun-Ade) [reply src/components/userDashboard/components/Tables.jsx]: ok

**Pull request merged**